testdata/dnn/tensorflow/gen_mirror_pad_testdata.py

```diff
@@ -4,14 +4,31 @@
 
 import numpy as np
 import cv2
+import tensorflow as tf
+
+def ReflectionPad2d(input, padding=1):
+    return tf.pad(
+      input, [[0, 0], [padding, padding], [padding, padding], [0, 0]],
+      mode='REFLECT')
 
 def main():
     inp = np.random.standard_normal([1, 3, 256, 256]).astype(np.float32)
     np.save('./mirror_pad_in.npy', inp)
-    net = cv2.dnn.readNet('./mirrorpad_net.pb')
-    net.setInput(inp)
-    outp = net.forward()
+
+    inp = inp.transpose(0, 2, 3, 1)
+    X_content = tf.placeholder(tf.float32, shape=(1, None, None, 3), name='X_content')
+    net = ReflectionPad2d(X_content)
+    net = tf.identity(net, 'finnal')
+
+    with tf.Session() as sess:
+        inference_graph = tf.graph_util.convert_variables_to_constants(sess, sess.graph_def, ['finnal'])
+        with open('./mirror_pad_net.pb', 'wb') as f:
+            f.write(inference_graph.SerializeToString())
+
+        outp = sess.run(net, feed_dict={X_content:inp})
+
+    outp = outp.transpose(0, 3, 1, 2)
     np.save('./mirror_pad_out.npy', outp)
 
 if __name__ == '__main__':
-  main()
\ No newline at end of file
+  main() 
```

Comment:
Please add it to an existing script: [generate_tf_models.py](https://github.com/opencv/opencv_extra/blob/master/testdata/dnn/tensorflow/generate_tf_models.py).